articles/controllers.js

```diff
@@ -1,13 +1,27 @@
-const articles = require('../db/data/test-data/articles')
-const {selectArticleById} = require('./models')
+const {selectArticleById, selectArticles} = require('./models')
 
 
 exports.getArticleById = (request, response, next) => {
     const { article_id } = request.params
 
-    selectArticleById(article_id)
-    .then((article) => {
-        response.status(200).send({ article })
+    if(/^\d+$/.test(article_id)) {
```

Comment:
This is a fine way to handle the bad request.

For your information: We could choose to let this request go through to the model and get a PSQL generated error, but this way of handling the error is also fine

Reply:
Thank you :)